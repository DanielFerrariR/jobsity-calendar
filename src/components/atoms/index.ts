export { default as Box } from './box'
export { default as Button } from './button'
export { default as Paper } from './paper'
export { default as Typography } from './typography'
export { default as TextField } from './text_field'
export { default as Container } from './container'
export { default as AppBar } from './appbar'
export { default as IconButton } from './icon_button'
export { default as Toolbar } from './toolbar'
export { default as Image } from './image'
export { default as Grid } from './grid'
export { default as Link } from './link'
export { default as Checkbox } from './checkbox'
export { default as Accordion } from './accordion'
export { default as AccordionSummary } from './accordion_summary'
export { default as AccordionDetails } from './accordion_details'
export { default as Tooltip } from './tooltip'
export { default as Modal } from './modal'
export { default as MenuItem } from './menu_item'
export { default as ListItemIcon } from './list_item_icon'
export { default as ListItemText } from './list_item_text'
export { default as ListItemSecondaryAction } from './list_item_secondary_action'
export { default as MenuList } from './menu_list'
export { default as Popover } from './popover'
export { default as Fade } from './fade'
export { default as Backdrop } from './backdrop'
export { default as CircularProgress } from './circular_progress'
export { default as LinearProgress } from './linear_progress'
export { default as Skeleton } from './skeleton'
export { default as List } from './list'
export { default as ListItem } from './list_item'
export { default as ClickAwayListener } from './click_away_listener'
export { default as Badge } from './badge'
export { default as Portal } from './portal'
export { default as Snackbar } from './snackbar'
export { default as Step } from './step'
export { default as StepContent } from './stepContent'
export { default as StepLabel } from './stepLabel'
export { default as Stepper } from './stepper'
export { default as TableBody } from './table_body'
export { default as TableContainer } from './table_container'
export { default as TableHead } from './table_head'
export { default as TableRow } from './table_row'
export { default as Table } from './table'
export { default as TableCell } from './table_cell'
export { default as DateTimePicker } from './date_time_picker'
