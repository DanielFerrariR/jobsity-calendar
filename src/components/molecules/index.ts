export { CustomTableHead, CustomTableBody } from './custom_table'
export { default as Modal } from './modal'
export { default as ReminderCard } from './reminder_card'
