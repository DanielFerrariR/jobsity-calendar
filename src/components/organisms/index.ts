export { default as Calendar } from './calendar'
export { default as ListRemindersModal } from './list_reminders_modal'
export { default as CreateReminderModal } from './create_reminder_modal'
export { default as EditReminderModal } from './edit_reminder_modal'
