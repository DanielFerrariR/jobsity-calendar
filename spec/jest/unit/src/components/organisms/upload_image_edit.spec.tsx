import React from 'react'
import { UploadImageEdit } from 'src/components/organisms'
import { render } from 'spec/jest/config/test_utils'
import { axe } from 'jest-axe'
import { imageData } from 'spec/jest/fixtures'

describe('testing UploadImageEdit component', () => {
  test('should snapshot works with english language', () => {
    const { container } = render(
      <UploadImageEdit
        imageData={imageData}
        setImageData={(): void => {}}
        imageFileFromServer=""
        setDeleteImageFromServer={(): void => {}}
      />
    )

    expect(container).toMatchSnapshot()
  })

  test('should snapshot works with portuguese language', () => {
    const { container } = render(
      <UploadImageEdit
        imageData={imageData}
        setImageData={(): void => {}}
        imageFileFromServer=""
        setDeleteImageFromServer={(): void => {}}
      />
    )

    expect(container).toMatchSnapshot()
  })

  test('should snapshot works with spanish language', () => {
    const { container } = render(
      <UploadImageEdit
        imageData={imageData}
        setImageData={(): void => {}}
        imageFileFromServer=""
        setDeleteImageFromServer={(): void => {}}
      />
    )

    expect(container).toMatchSnapshot()
  })

  test('should the page be accessible', async () => {
    const { container } = render(
      <UploadImageEdit
        imageData={imageData}
        setImageData={(): void => {}}
        imageFileFromServer=""
        setDeleteImageFromServer={(): void => {}}
      />
    )

    const results = await axe(container)

    expect(results).toHaveNoViolations()
  })
})
