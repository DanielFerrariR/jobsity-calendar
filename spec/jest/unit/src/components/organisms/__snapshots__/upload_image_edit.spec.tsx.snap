// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing UploadImageEdit component should snapshot works with english language 1`] = `
<div>
  <label
    class="WithStyles(ForwardRef(Tooltip))-root WithStyles(ForwardRef(Tooltip))-root"
    for="input-create-credential-upload-file"
    title=""
  >
    <input
      accept=".png, .jpg, .jpeg"
      data-testid="input-upload-image-edit"
      id="input-create-credential-upload-file"
      style="display: none;"
      type="file"
    />
    <span
      aria-disabled="false"
      class="MuiButtonBase-root MuiButton-root MuiButton-contained Component-root Component-root MuiButton-containedPrimary"
      role="button"
      tabindex="0"
    >
      <span
        class="MuiButton-label"
      >
        Upload Image
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </span>
  </label>
  <div
    class="MuiBox-root MuiBox-root Component-root Component-root"
  >
    <h6
      class="MuiTypography-root Component-root Component-root MuiTypography-subtitle1"
    >
      New Image: Save to Send
    </h6>
    <div
      class="MuiBox-root MuiBox-root Component-root Component-root"
    >
      <button
        aria-label="Download"
        class="MuiButtonBase-root MuiIconButton-root Component-root Component-root Styled(Component)-root Styled(Component)-root WithStyles(ForwardRef(Tooltip))-root WithStyles(ForwardRef(Tooltip))-root"
        tabindex="0"
        title="Download"
        type="button"
      >
        <span
          class="MuiIconButton-label"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-colorSecondary"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 9h-4V3H9v6H5l7 7 7-7zM5 18v2h14v-2H5z"
            />
          </svg>
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
      <button
        aria-label="Close"
        class="MuiButtonBase-root MuiIconButton-root Component-root Component-root Styled(Component)-root Styled(Component)-root WithStyles(ForwardRef(Tooltip))-root WithStyles(ForwardRef(Tooltip))-root"
        data-testid="tooltip-upload-image-edit-close"
        tabindex="0"
        title="Close"
        type="button"
      >
        <span
          class="MuiIconButton-label"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-colorSecondary"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
            />
          </svg>
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </div>
  </div>
  <div
    class="MuiBox-root MuiBox-root Component-root Component-root"
  >
    <img
      alt="Image"
      class="img-root img-root"
      data-testid="image-upload-edit"
      src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAALsAAADIBAMAAABCCI02AAAALVBMVEX///89UmtXa4JNWWtHY4I0NDT29/jEx8vZ293r7O1+hpCVm6OtsrhKSktvb3HCtbr0AAAI1ElEQVR42u2a+28UVRTHbzCZznQ3JAMkmsIPdtvI85dlrQU1JoVJNMBvm4tYmM1GjJS2mLWA5VHMikIrBTMpBmlRAy1FRTFVXIG2kirQ8tL4yyjQlBCS+gD0b/A+5r2zj7mz609zAls6M/uZO+d8z7nn3gGAwAILLLDAAgsssMDKbkJX75GSApsU628zIpHTX+4o3R34gSEHHtnpq8nS0LnL4kvZ+EjkK6Uk+AOiWOWGjyxOl4AejorinLQbvhR87oSI7JTlQE/E5Pv2Tx+mi49rmuwY7r8fsdiQX9VUE/wsPExhdySGbKkFn/KJ3yZSGwNg00As5sAv8jv4qIZfCI7VxLLwC3zit2p0cW5rLJaNHyrR4MXZNW54n8LZKxo22wW/xGdtPJEff8ofvlm0mAvepyx/zI9flCxRYG3eWVoiWb4q5sf7dP03NryYhV/tq2DyDTnwpym9VppfMt1YvFPfe3R3P8FLF3zg33Tg9eHX48n9eiQiSdIqxX+xdMMDrhvjJR/i4c+6e6eenm7D+BU+Mos/kQ+/DeN9RXdTtZt3KF6olYzhs2Zva0NOfFiithDdaYR1/Mdc6gLF79Xwq5KgLcacX33Z+GXkxHcaXhrqi8WeKsWMIlaZeL5Wx69G89jz7PFtM+izUmcNfLNOl0idYC/NnCH/L0CPgd+m01eSJxpjH36PFt65SRDW8dxJGz32JDv+skTxqOsQamKLJ/ZbZKnRtXAzzecoiGTwJBDnqZd7LI7H9ix7A44xepO8WV9R2Ok+YvsqcYK9xddkudLAx5hL23uE1DDPeizkpMeY20EtPddYH7/H7ho/0tHT8zNruE866LHlrMIx0vO85WiLJA3Y8PV+hENL+wXr8Of32fCsVSckmfxx83C3UmHDswp/homXVttq0UCp8S/azlyy4pcy4j+04Bfazrxrwyf9ZBW14/Z0tjlf8Y+3p2albfRe8dzrxPC8EVlRS/6M5cbHjpCrPQjeWOPM79XM7l+h92i3o4+bk2TB577m/8XPLh5fUcTyNeTAz2LBL855TaUD70H9fBH7Hq858FXF4wUTn1PT7zjxHnTfb/DTRa2rkc3zgP/WwOfswarZ8cDEH88VHufq6wkvvWvOTSch5S4c0cuM25ND+EJnv3a/w068lz6/OeK+9TGjFi9JnPs9ehvKUBUiKSCM2+avlPva0UsfbuZV5BRoIx0s6TEx/pT72tHLGkswhb+kTxvZpfMUv8aQF2PJQfaTOfwGsnoAPO7v8XJ8haI/Ya0FP9fbqs1KxzulIITnpact8yIKf42Jf9wTvkWnU8LscdocPG3uVXD4AVca+IWe8HpJ1p+/av+AgafSpFdUs+jSjK1jXUvwVJptFt+hx/O4OKexjTo2FSj+C6t0o16rvZm31c5FP8WvsVWlqPfIUtfWZO2IUDySZqsp3Nqo/Q1IsSW/VsyBl4aEb62vOZD/PS/f9lodr+9GafgFu6z0Bd1Rr65HVa1BzIlfEbG9o+EOvOi5f/3JbSdQw0s2vAI4z3tGB8R8eOvwWXbsWqtdd2Hd8Cyr5m/EvHiLd5h28t9136J2wTNtxobd9+91/Ko2f285uBP58KvSYT+Bxa2I67sTil+SNttExp002wsCG37RVUs9Zn59tdXtvdVzNyZ22qYz5q2icDR78JZtOd6P520d9pzrLngy3yzy8YKmxegxuF012Xjchl4rxcsl1AQc7M/CI2l+7odueB9rr+nGfQee6/8s6QvPbTXflaMbdAxHbDuunf7oaL6tts3SXG/XW6CUdkBkmKU9brCnyobH7qlKlg8PWqPLQDmte6iseJAEgQUWWGCB5bLNxfx/y6au/YzN69TdwkV415Q89SdTra6EMGt63ez4vQ9i+5sFvwXCe84pfcq+W8pNQzmmwlss+CsQ3nbO6I4bhqC88/WPBv9gwU8n4IYC+C1k4ILCQBfUDXUJx7GQA78dMk/wYbg2I5O+vQN/cjcV0PQ2rG/HQxV2j1yleDP4m0ZGNYkeHDnX3t5eoOWfCe/tI4OrgGuJju6BfVgmKLj8IPpZb8cfUyGUyfrtZ6ImWCAgh+DYY+SabPwe8v0hfI0O4evwofV4+4rS4TP58Rk5GYbrbPiWs3ByJA14FT7qGMRx3wgbtbBuhPHR6yp+mAycnDgL5ZH8K1GurhEIZDgmnn6AN+DvaLiqrKD4wElaOTLwe/owghpHdxyEBRreCvgyyho56YLfDsc01XAoCImdRGaJJP0O/gvAJ/B4fnwI/oYzK+WCH5QVGnokLxzQcaBBBXU9Fpx+Mp+R+5OPLPx0ggoX5/THKKSJNBrMbVIjGtHhV4rBb4dpfNXtbDweI/HeOiL3M7ikVcLl+F0qunGRzpmOE2duyMbzKqkVPOGQJGhE44jjBaMaT6IoKIDLFAgtuqqrq6tTTbjhCVfQ8Mj/svKYJnYUYCTM9ov4H/lLAlTx9ags5Bi9gUfA1EwoT2Gr19NqWaGSoNlYAd8j2wfHZsLl5F20QuYACO8ohUrCC/8i+wclihOPE4483y0THzKnhk/h3ZFzhUr0FXk/lkIzYmQLE0cdPd8zhshSYcNTIBMvXP65OhobHpUF6oXDJp4kGxrzccAnidrlNK8asVTlwnhem6i4OiI1UhHv4VT+lYgauUWoQ4mxpz6JY4nOT5Pa3YeqRR28O1qoN6nUfYkFPA0ftZ8hFTYMGzvT6FM+1/kDUjs6c+fmDRU75hCMT3T+gu97BQviTv5Nh416UmMXkDI/SW6okuk9o88XGV1dyIPE1tEyhASdN6326VmHA1iBvpB4v+4WERQeYAWeOyaRX3gsQvgXvvADklXoW1thAt9gQz78x6NapPgbuGZefJgGHfiG3PUH+EfF8L/nyAXC7q8fXNW+MvxgVMHhQUXh4BkoK6AcRpsHYRqmy4IfJEmBvFse/DRNgEy5Rg8vkLYkkSyT7+MPRy6qeLYvh4VoAsTLtDXGvY1Vf/caKJcd7bi5QwFltWDrKrDAAgsssMACC6w09h+nVufMnQQOEgAAAABJRU5ErkJggg=="
    />
  </div>
</div>
`;

exports[`testing UploadImageEdit component should snapshot works with portuguese language 1`] = `
<div>
  <label
    class="WithStyles(ForwardRef(Tooltip))-root WithStyles(ForwardRef(Tooltip))-root"
    for="input-create-credential-upload-file"
    title=""
  >
    <input
      accept=".png, .jpg, .jpeg"
      data-testid="input-upload-image-edit"
      id="input-create-credential-upload-file"
      style="display: none;"
      type="file"
    />
    <span
      aria-disabled="false"
      class="MuiButtonBase-root MuiButton-root MuiButton-contained Component-root Component-root MuiButton-containedPrimary"
      role="button"
      tabindex="0"
    >
      <span
        class="MuiButton-label"
      >
        Upload Image
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </span>
  </label>
  <div
    class="MuiBox-root MuiBox-root Component-root Component-root"
  >
    <h6
      class="MuiTypography-root Component-root Component-root MuiTypography-subtitle1"
    >
      New Image: Save to Send
    </h6>
    <div
      class="MuiBox-root MuiBox-root Component-root Component-root"
    >
      <button
        aria-label="Download"
        class="MuiButtonBase-root MuiIconButton-root Component-root Component-root Styled(Component)-root Styled(Component)-root WithStyles(ForwardRef(Tooltip))-root WithStyles(ForwardRef(Tooltip))-root"
        tabindex="0"
        title="Download"
        type="button"
      >
        <span
          class="MuiIconButton-label"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-colorSecondary"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 9h-4V3H9v6H5l7 7 7-7zM5 18v2h14v-2H5z"
            />
          </svg>
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
      <button
        aria-label="Close"
        class="MuiButtonBase-root MuiIconButton-root Component-root Component-root Styled(Component)-root Styled(Component)-root WithStyles(ForwardRef(Tooltip))-root WithStyles(ForwardRef(Tooltip))-root"
        data-testid="tooltip-upload-image-edit-close"
        tabindex="0"
        title="Close"
        type="button"
      >
        <span
          class="MuiIconButton-label"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-colorSecondary"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
            />
          </svg>
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </div>
  </div>
  <div
    class="MuiBox-root MuiBox-root Component-root Component-root"
  >
    <img
      alt="Image"
      class="img-root img-root"
      data-testid="image-upload-edit"
      src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAALsAAADIBAMAAABCCI02AAAALVBMVEX///89UmtXa4JNWWtHY4I0NDT29/jEx8vZ293r7O1+hpCVm6OtsrhKSktvb3HCtbr0AAAI1ElEQVR42u2a+28UVRTHbzCZznQ3JAMkmsIPdtvI85dlrQU1JoVJNMBvm4tYmM1GjJS2mLWA5VHMikIrBTMpBmlRAy1FRTFVXIG2kirQ8tL4yyjQlBCS+gD0b/A+5r2zj7mz609zAls6M/uZO+d8z7nn3gGAwAILLLDAAgsssMDKbkJX75GSApsU628zIpHTX+4o3R34gSEHHtnpq8nS0LnL4kvZ+EjkK6Uk+AOiWOWGjyxOl4AejorinLQbvhR87oSI7JTlQE/E5Pv2Tx+mi49rmuwY7r8fsdiQX9VUE/wsPExhdySGbKkFn/KJ3yZSGwNg00As5sAv8jv4qIZfCI7VxLLwC3zit2p0cW5rLJaNHyrR4MXZNW54n8LZKxo22wW/xGdtPJEff8ofvlm0mAvepyx/zI9flCxRYG3eWVoiWb4q5sf7dP03NryYhV/tq2DyDTnwpym9VppfMt1YvFPfe3R3P8FLF3zg33Tg9eHX48n9eiQiSdIqxX+xdMMDrhvjJR/i4c+6e6eenm7D+BU+Mos/kQ+/DeN9RXdTtZt3KF6olYzhs2Zva0NOfFiithDdaYR1/Mdc6gLF79Xwq5KgLcacX33Z+GXkxHcaXhrqi8WeKsWMIlaZeL5Wx69G89jz7PFtM+izUmcNfLNOl0idYC/NnCH/L0CPgd+m01eSJxpjH36PFt65SRDW8dxJGz32JDv+skTxqOsQamKLJ/ZbZKnRtXAzzecoiGTwJBDnqZd7LI7H9ix7A44xepO8WV9R2Ok+YvsqcYK9xddkudLAx5hL23uE1DDPeizkpMeY20EtPddYH7/H7ho/0tHT8zNruE866LHlrMIx0vO85WiLJA3Y8PV+hENL+wXr8Of32fCsVSckmfxx83C3UmHDswp/homXVttq0UCp8S/azlyy4pcy4j+04Bfazrxrwyf9ZBW14/Z0tjlf8Y+3p2albfRe8dzrxPC8EVlRS/6M5cbHjpCrPQjeWOPM79XM7l+h92i3o4+bk2TB577m/8XPLh5fUcTyNeTAz2LBL855TaUD70H9fBH7Hq858FXF4wUTn1PT7zjxHnTfb/DTRa2rkc3zgP/WwOfswarZ8cDEH88VHufq6wkvvWvOTSch5S4c0cuM25ND+EJnv3a/w068lz6/OeK+9TGjFi9JnPs9ehvKUBUiKSCM2+avlPva0UsfbuZV5BRoIx0s6TEx/pT72tHLGkswhb+kTxvZpfMUv8aQF2PJQfaTOfwGsnoAPO7v8XJ8haI/Ya0FP9fbqs1KxzulIITnpact8yIKf42Jf9wTvkWnU8LscdocPG3uVXD4AVca+IWe8HpJ1p+/av+AgafSpFdUs+jSjK1jXUvwVJptFt+hx/O4OKexjTo2FSj+C6t0o16rvZm31c5FP8WvsVWlqPfIUtfWZO2IUDySZqsp3Nqo/Q1IsSW/VsyBl4aEb62vOZD/PS/f9lodr+9GafgFu6z0Bd1Rr65HVa1BzIlfEbG9o+EOvOi5f/3JbSdQw0s2vAI4z3tGB8R8eOvwWXbsWqtdd2Hd8Cyr5m/EvHiLd5h28t9136J2wTNtxobd9+91/Ko2f285uBP58KvSYT+Bxa2I67sTil+SNttExp002wsCG37RVUs9Zn59tdXtvdVzNyZ22qYz5q2icDR78JZtOd6P520d9pzrLngy3yzy8YKmxegxuF012Xjchl4rxcsl1AQc7M/CI2l+7odueB9rr+nGfQee6/8s6QvPbTXflaMbdAxHbDuunf7oaL6tts3SXG/XW6CUdkBkmKU9brCnyobH7qlKlg8PWqPLQDmte6iseJAEgQUWWGCB5bLNxfx/y6au/YzN69TdwkV415Q89SdTra6EMGt63ez4vQ9i+5sFvwXCe84pfcq+W8pNQzmmwlss+CsQ3nbO6I4bhqC88/WPBv9gwU8n4IYC+C1k4ILCQBfUDXUJx7GQA78dMk/wYbg2I5O+vQN/cjcV0PQ2rG/HQxV2j1yleDP4m0ZGNYkeHDnX3t5eoOWfCe/tI4OrgGuJju6BfVgmKLj8IPpZb8cfUyGUyfrtZ6ImWCAgh+DYY+SabPwe8v0hfI0O4evwofV4+4rS4TP58Rk5GYbrbPiWs3ByJA14FT7qGMRx3wgbtbBuhPHR6yp+mAycnDgL5ZH8K1GurhEIZDgmnn6AN+DvaLiqrKD4wElaOTLwe/owghpHdxyEBRreCvgyyho56YLfDsc01XAoCImdRGaJJP0O/gvAJ/B4fnwI/oYzK+WCH5QVGnokLxzQcaBBBXU9Fpx+Mp+R+5OPLPx0ggoX5/THKKSJNBrMbVIjGtHhV4rBb4dpfNXtbDweI/HeOiL3M7ikVcLl+F0qunGRzpmOE2duyMbzKqkVPOGQJGhE44jjBaMaT6IoKIDLFAgtuqqrq6tTTbjhCVfQ8Mj/svKYJnYUYCTM9ov4H/lLAlTx9ags5Bi9gUfA1EwoT2Gr19NqWaGSoNlYAd8j2wfHZsLl5F20QuYACO8ohUrCC/8i+wclihOPE4483y0THzKnhk/h3ZFzhUr0FXk/lkIzYmQLE0cdPd8zhshSYcNTIBMvXP65OhobHpUF6oXDJp4kGxrzccAnidrlNK8asVTlwnhem6i4OiI1UhHv4VT+lYgauUWoQ4mxpz6JY4nOT5Pa3YeqRR28O1qoN6nUfYkFPA0ftZ8hFTYMGzvT6FM+1/kDUjs6c+fmDRU75hCMT3T+gu97BQviTv5Nh416UmMXkDI/SW6okuk9o88XGV1dyIPE1tEyhASdN6326VmHA1iBvpB4v+4WERQeYAWeOyaRX3gsQvgXvvADklXoW1thAt9gQz78x6NapPgbuGZefJgGHfiG3PUH+EfF8L/nyAXC7q8fXNW+MvxgVMHhQUXh4BkoK6AcRpsHYRqmy4IfJEmBvFse/DRNgEy5Rg8vkLYkkSyT7+MPRy6qeLYvh4VoAsTLtDXGvY1Vf/caKJcd7bi5QwFltWDrKrDAAgsssMACC6w09h+nVufMnQQOEgAAAABJRU5ErkJggg=="
    />
  </div>
</div>
`;

exports[`testing UploadImageEdit component should snapshot works with spanish language 1`] = `
<div>
  <label
    class="WithStyles(ForwardRef(Tooltip))-root WithStyles(ForwardRef(Tooltip))-root"
    for="input-create-credential-upload-file"
    title=""
  >
    <input
      accept=".png, .jpg, .jpeg"
      data-testid="input-upload-image-edit"
      id="input-create-credential-upload-file"
      style="display: none;"
      type="file"
    />
    <span
      aria-disabled="false"
      class="MuiButtonBase-root MuiButton-root MuiButton-contained Component-root Component-root MuiButton-containedPrimary"
      role="button"
      tabindex="0"
    >
      <span
        class="MuiButton-label"
      >
        Upload Image
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </span>
  </label>
  <div
    class="MuiBox-root MuiBox-root Component-root Component-root"
  >
    <h6
      class="MuiTypography-root Component-root Component-root MuiTypography-subtitle1"
    >
      New Image: Save to Send
    </h6>
    <div
      class="MuiBox-root MuiBox-root Component-root Component-root"
    >
      <button
        aria-label="Download"
        class="MuiButtonBase-root MuiIconButton-root Component-root Component-root Styled(Component)-root Styled(Component)-root WithStyles(ForwardRef(Tooltip))-root WithStyles(ForwardRef(Tooltip))-root"
        tabindex="0"
        title="Download"
        type="button"
      >
        <span
          class="MuiIconButton-label"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-colorSecondary"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 9h-4V3H9v6H5l7 7 7-7zM5 18v2h14v-2H5z"
            />
          </svg>
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
      <button
        aria-label="Close"
        class="MuiButtonBase-root MuiIconButton-root Component-root Component-root Styled(Component)-root Styled(Component)-root WithStyles(ForwardRef(Tooltip))-root WithStyles(ForwardRef(Tooltip))-root"
        data-testid="tooltip-upload-image-edit-close"
        tabindex="0"
        title="Close"
        type="button"
      >
        <span
          class="MuiIconButton-label"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-colorSecondary"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
            />
          </svg>
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </div>
  </div>
  <div
    class="MuiBox-root MuiBox-root Component-root Component-root"
  >
    <img
      alt="Image"
      class="img-root img-root"
      data-testid="image-upload-edit"
      src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAALsAAADIBAMAAABCCI02AAAALVBMVEX///89UmtXa4JNWWtHY4I0NDT29/jEx8vZ293r7O1+hpCVm6OtsrhKSktvb3HCtbr0AAAI1ElEQVR42u2a+28UVRTHbzCZznQ3JAMkmsIPdtvI85dlrQU1JoVJNMBvm4tYmM1GjJS2mLWA5VHMikIrBTMpBmlRAy1FRTFVXIG2kirQ8tL4yyjQlBCS+gD0b/A+5r2zj7mz609zAls6M/uZO+d8z7nn3gGAwAILLLDAAgsssMDKbkJX75GSApsU628zIpHTX+4o3R34gSEHHtnpq8nS0LnL4kvZ+EjkK6Uk+AOiWOWGjyxOl4AejorinLQbvhR87oSI7JTlQE/E5Pv2Tx+mi49rmuwY7r8fsdiQX9VUE/wsPExhdySGbKkFn/KJ3yZSGwNg00As5sAv8jv4qIZfCI7VxLLwC3zit2p0cW5rLJaNHyrR4MXZNW54n8LZKxo22wW/xGdtPJEff8ofvlm0mAvepyx/zI9flCxRYG3eWVoiWb4q5sf7dP03NryYhV/tq2DyDTnwpym9VppfMt1YvFPfe3R3P8FLF3zg33Tg9eHX48n9eiQiSdIqxX+xdMMDrhvjJR/i4c+6e6eenm7D+BU+Mos/kQ+/DeN9RXdTtZt3KF6olYzhs2Zva0NOfFiithDdaYR1/Mdc6gLF79Xwq5KgLcacX33Z+GXkxHcaXhrqi8WeKsWMIlaZeL5Wx69G89jz7PFtM+izUmcNfLNOl0idYC/NnCH/L0CPgd+m01eSJxpjH36PFt65SRDW8dxJGz32JDv+skTxqOsQamKLJ/ZbZKnRtXAzzecoiGTwJBDnqZd7LI7H9ix7A44xepO8WV9R2Ok+YvsqcYK9xddkudLAx5hL23uE1DDPeizkpMeY20EtPddYH7/H7ho/0tHT8zNruE866LHlrMIx0vO85WiLJA3Y8PV+hENL+wXr8Of32fCsVSckmfxx83C3UmHDswp/homXVttq0UCp8S/azlyy4pcy4j+04Bfazrxrwyf9ZBW14/Z0tjlf8Y+3p2albfRe8dzrxPC8EVlRS/6M5cbHjpCrPQjeWOPM79XM7l+h92i3o4+bk2TB577m/8XPLh5fUcTyNeTAz2LBL855TaUD70H9fBH7Hq858FXF4wUTn1PT7zjxHnTfb/DTRa2rkc3zgP/WwOfswarZ8cDEH88VHufq6wkvvWvOTSch5S4c0cuM25ND+EJnv3a/w068lz6/OeK+9TGjFi9JnPs9ehvKUBUiKSCM2+avlPva0UsfbuZV5BRoIx0s6TEx/pT72tHLGkswhb+kTxvZpfMUv8aQF2PJQfaTOfwGsnoAPO7v8XJ8haI/Ya0FP9fbqs1KxzulIITnpact8yIKf42Jf9wTvkWnU8LscdocPG3uVXD4AVca+IWe8HpJ1p+/av+AgafSpFdUs+jSjK1jXUvwVJptFt+hx/O4OKexjTo2FSj+C6t0o16rvZm31c5FP8WvsVWlqPfIUtfWZO2IUDySZqsp3Nqo/Q1IsSW/VsyBl4aEb62vOZD/PS/f9lodr+9GafgFu6z0Bd1Rr65HVa1BzIlfEbG9o+EOvOi5f/3JbSdQw0s2vAI4z3tGB8R8eOvwWXbsWqtdd2Hd8Cyr5m/EvHiLd5h28t9136J2wTNtxobd9+91/Ko2f285uBP58KvSYT+Bxa2I67sTil+SNttExp002wsCG37RVUs9Zn59tdXtvdVzNyZ22qYz5q2icDR78JZtOd6P520d9pzrLngy3yzy8YKmxegxuF012Xjchl4rxcsl1AQc7M/CI2l+7odueB9rr+nGfQee6/8s6QvPbTXflaMbdAxHbDuunf7oaL6tts3SXG/XW6CUdkBkmKU9brCnyobH7qlKlg8PWqPLQDmte6iseJAEgQUWWGCB5bLNxfx/y6au/YzN69TdwkV415Q89SdTra6EMGt63ez4vQ9i+5sFvwXCe84pfcq+W8pNQzmmwlss+CsQ3nbO6I4bhqC88/WPBv9gwU8n4IYC+C1k4ILCQBfUDXUJx7GQA78dMk/wYbg2I5O+vQN/cjcV0PQ2rG/HQxV2j1yleDP4m0ZGNYkeHDnX3t5eoOWfCe/tI4OrgGuJju6BfVgmKLj8IPpZb8cfUyGUyfrtZ6ImWCAgh+DYY+SabPwe8v0hfI0O4evwofV4+4rS4TP58Rk5GYbrbPiWs3ByJA14FT7qGMRx3wgbtbBuhPHR6yp+mAycnDgL5ZH8K1GurhEIZDgmnn6AN+DvaLiqrKD4wElaOTLwe/owghpHdxyEBRreCvgyyho56YLfDsc01XAoCImdRGaJJP0O/gvAJ/B4fnwI/oYzK+WCH5QVGnokLxzQcaBBBXU9Fpx+Mp+R+5OPLPx0ggoX5/THKKSJNBrMbVIjGtHhV4rBb4dpfNXtbDweI/HeOiL3M7ikVcLl+F0qunGRzpmOE2duyMbzKqkVPOGQJGhE44jjBaMaT6IoKIDLFAgtuqqrq6tTTbjhCVfQ8Mj/svKYJnYUYCTM9ov4H/lLAlTx9ags5Bi9gUfA1EwoT2Gr19NqWaGSoNlYAd8j2wfHZsLl5F20QuYACO8ohUrCC/8i+wclihOPE4483y0THzKnhk/h3ZFzhUr0FXk/lkIzYmQLE0cdPd8zhshSYcNTIBMvXP65OhobHpUF6oXDJp4kGxrzccAnidrlNK8asVTlwnhem6i4OiI1UhHv4VT+lYgauUWoQ4mxpz6JY4nOT5Pa3YeqRR28O1qoN6nUfYkFPA0ftZ8hFTYMGzvT6FM+1/kDUjs6c+fmDRU75hCMT3T+gu97BQviTv5Nh416UmMXkDI/SW6okuk9o88XGV1dyIPE1tEyhASdN6326VmHA1iBvpB4v+4WERQeYAWeOyaRX3gsQvgXvvADklXoW1thAt9gQz78x6NapPgbuGZefJgGHfiG3PUH+EfF8L/nyAXC7q8fXNW+MvxgVMHhQUXh4BkoK6AcRpsHYRqmy4IfJEmBvFse/DRNgEy5Rg8vkLYkkSyT7+MPRy6qeLYvh4VoAsTLtDXGvY1Vf/caKJcd7bi5QwFltWDrKrDAAgsssMACC6w09h+nVufMnQQOEgAAAABJRU5ErkJggg=="
    />
  </div>
</div>
`;
